include/tss2/tss2_tcti.h

```diff
@@ -24,6 +49,9 @@ typedef struct pollfd TSS2_TCTI_POLL_HANDLE;
 #elif defined(_WIN32)
 #include <windows.h>
 typedef HANDLE TSS2_TCTI_POLL_HANDLE;
+#elif defined (__VXWORKS__)
+#include <sys/poll.h>
```

Comment:
I'm not sure if this shouldn't be done by adding the this directory to the include-dirs ?

Why does vxworks have it at this place anyways ?

Reply:
The sys directory includes some header files which are duplicated with other header files in include-dirs. Therefore, sys can't be added to include-dirs.

Reply:
You have 2 `poll.h` files ?
And the function that POSIX.1-2001 defines for `poll.h` you've put into `sys/poll.h` instead ?

Reply:
Only one poll.h, but there are two errno.h files.